Tests/Functional/Domain/Repository/CurrentWeatherRepositoryTest.php

```diff
@@ -26,30 +25,13 @@ class CurrentWeatherRepositoryTest extends FunctionalTestCase
      */
     protected $subject;
 
-    /**
-     * @var string[]
-     */
-    protected $testExtensionsToLoad = [
-        'typo3conf/ext/weather2',
-        'typo3conf/ext/static_info_tables',
-    ];
+    protected array $testExtensionsToLoad = ['typo3conf/ext/weather2'];
```

Comment:
Since testingframework 7.0.3 is has to the package name. And please wrap it into multiple lines.